commands/ThresholdCommand.java

```diff
@@ -0,0 +1,44 @@
+package org.usfirst.frc4904.standard.commands;
+
+
+import java.util.function.Supplier;
+import edu.wpi.first.wpilibj.command.Command;
+
+/**
+ *
+ */
+public class ThresholdCommand extends Command {
+	protected final Command command;
+	protected final Supplier<Double> axis;
+	protected final double threshold;
+	protected final boolean invert;
+
+	public ThresholdCommand(Command command, Supplier<Double> axis, double threshold, boolean invert) {
```

Comment:
Is it worth it to make this work with other comparable types, not just `Double`? (i.e. `Integer`) I'm not sure.